docs/firebase.md

```diff
@@ -308,3 +308,23 @@ app:
 Open the web app by visiting the URL printed by the `deploy` command. The app
 requires you to sign in with a Google account, after which you can initiate
 endpoint requests.
+
+## Developing using Firebase Local Emulator Suite
+
+Firebase offers a suite of emualtors for local development and you can use Genkit with [Firebase Local Emulator Suite](https://firebase.google.com/docs/emulator-suite).
+
+To use Genkit with Firebase emulator start the emulator like this:
+
+```bash
+GENKIT_ENV=dev firebase emulators:start --inspect-functions
```

Comment:
Is this sufficient? I've got the following:

```
"serve-genkit": "npm run build && GENKIT_ENV=dev GENKIT_REFLECTION_ON_STARTUP_FAILURE=ignore firebase emulators:start --inspect-functions"
```

I haven't tried which is the absolute essential number of parameters to make it work...

Reply:
We made a few changes, `GENKIT_REFLECTION_ON_STARTUP_FAILURE` is not necessary now.